libpod/image/pull.go

```diff
@@ -177,8 +176,12 @@ func (i *Image) pullImage(writer io.Writer, authfile, signaturePolicyPath string
 	copyOptions := getCopyOptions(writer, signaturePolicyPath, dockerOptions, nil, signingOptions, authfile, "", false)
 	for _, imageInfo := range pullStructs {
 		// Print the following statement only when pulling from a docker or atomic registry
-		if writer != nil && (imageInfo.srcRef.Transport().Name() == DockerTransport || imageInfo.srcRef.Transport().Name() == AtomicTransport) {
-			io.WriteString(writer, fmt.Sprintf("Trying to pull %s...\n", imageInfo.image))
+		foo := imageInfo.srcRef.Transport().Name()
```

Comment:
Is this necessary? These 4 lines seem like they do nothing...

Reply:
leftover debug